code/engine/xrCore/_math.h

```diff
@@ -31,10 +31,4 @@ inline u64 GetCLK() { return __rdtsc(); }
 } // namespace CPU
 
 extern XRCORE_API void _initialize_cpu();
-extern XRCORE_API void _initialize_cpu_thread();
-
-// threading
-typedef void thread_t(void*);
-extern XRCORE_API void thread_name(const char* name);
-extern XRCORE_API void thread_spawn(thread_t* entry, const char* name, unsigned stack,
-                                    void* arglist);
+extern XRCORE_API void set_thread_name(std::wstring_view name);
```

Comment:
It should be called set_current_thread_name. Also, it would be nice to add set_thread_name function which takes a thread handle argument